src/main/scala/jtag/JtagTap.scala

```diff
@@ -79,15 +82,19 @@ class JtagTapController(irLength: Int, initialInstruction: BigInt)(implicit val
   // combined with any POR, and it should also be
   // synchronized to TCK.
   require(!io.jtag.TRSTn.isDefined, "TRSTn should be absorbed into jtckPOReset outside of JtagTapController.")
-  withReset(io.control.jtag_reset) {
+  withReset(io.control.jtag_reset.asAsyncReset) {
```

Comment:
we could get rid of the cast here if we know that it's gotta be AsyncReset already

Reply:
Done.